Dockerfile

```diff
@@ -99,12 +99,13 @@ RUN echo "installing sdk tools" && \
 # Install SDKs
 # Please keep these in descending order!
 # The `yes` is for accepting all non-standard tool licenses.
-    mkdir --parents "$HOME/.android/" && \
+    mkdir --parents "$ANDROID_HOME/.android/" && \
```

Comment:
Have you test this ? This suppose to user's home directory to keep sdk settings.